### DISCUSSION TITLE
nvim-treesitter Error

### DISCUSSION THREAD

**Discussion by ChunChoc** [NvChad/NvChad#1812] [Q&A] [answered]
Hi! :D
I had this error during the installation of NvChad
Any idea how to solve it? I would appreciate your help
Sorry, I'm a beginner...
I'm using Fedora bdw 

![nvchad_error](https://user-images.githubusercontent.com/127690040/224567594-f96fd315-8bad-4058-9cf1-5cda76936533.png)

**Comment by siduck** [comment, marked as answer]
@ChunChoc https://github.com/nvim-treesitter/nvim-treesitter/issues/2758

https://github.com/nvim-treesitter/nvim-treesitter/search?q=cc1plus&type=issues

**Reply by ChunChoc** [reply]
Thanks!
